scenarios/classification/README.md

```diff
@@ -2,6 +2,10 @@
 
 This directory provides examples and best practices for building image classification systems. Our goal is to enable users to easily and quickly train high-accuracy classifiers on their own datasets. We provide example notebooks with pre-set default parameters that are shown to work well on a variety of data sets. We also include extensive documentation of common pitfalls and best practices. Additionally, we show how Azure, Microsoft's cloud computing platform, can be used to speed up training on large data sets or deploy models as web services.
 
+| Image classification (single object) | Image classification (multiple objects) |
+|--|--|
+| <img align="center" src="./media/ic_example2.jpg" height="200"/>  | <img align="center" src="./media/ic_example3.jpg" height="200"/> |
```

Comment:
[nit] is there a specific reason why we're starting from ic_example2, instead of ic_example1?

Reply:
No, there isn't a good reason other that I had multiple jpg's to choose from.